solidity/test/BancorConverterUpgrader.js

```diff
@@ -184,10 +179,10 @@ contract('BancorConverterUpgrader', accounts => {
         describe(`${init.name}(version = ${version ? version : 'latest'}, active = ${active}):`, () => {
             it('upgrade should complete successfully', async () => {
                 const [upgrader, oldConverter] = await init(deployer, version, active);
-                const oldConverterInitialState = getConverterState(oldConverter);
+                const oldConverterInitialState = await getConverterState(oldConverter);
```

Comment:
@yudilevi : Hell with this @#$%. How long has that been missing there?

Reply:
since it was created, apparently

Reply:
@yudilevi: No, only since my huge refactoring of this test on 22/03 this year.

Reply:
so there you go